src/components/profile/ReportingStructure.vue

```diff
@@ -49,6 +54,52 @@
         </ShowMore>
       </div>
     </div>
+    <div
```

Comment:
That's basically the same as above. We could de-duplicate some code here by moving that into a component.

Reply:
Yea I was thinking about that and I thought I'd do that in a later task. I'll get that done here though.